hyperion2boblight/lib/boblight_client.py

```diff
@@ -30,10 +30,28 @@ def __init__(self, server_address, priority_list):
             raise socket_error
 
         self.socket.sendall(bytes('hello\n', "utf8"))
```

Comment:
Can you please move the 'hello <--> hello back' checking in a separate function?
Then update test_boblight_client.py:test_boblight_client_create() to reflect the changes.
Finally, update boblight_client.py:run() accordingly.

Many thanks for the PR.